packages/web-components/fast-ssr/src/template-renderer/events.ts

```diff
@@ -0,0 +1,24 @@
+export const TemplateRendererEventTypes = {
```

Comment:
If I recall correctly, these events were primarily added to aid in debugging. Emitting events does have some runtime cost, so maybe consider not including this part if these events are no longer necessary.

Reply:
I have removed those events.